src/styles/days_settings.scss

```diff
@@ -0,0 +1,33 @@
+// довелось вставити цю команду, бо prettier не дає розмістити
+// $days в один рядок, а після переносу не бачить пропуску перед двокрапкою
+
+/* prettier-ignore */
+$days: 'mon' 0,'tue' 1,'wed' 2,'thu' 3,'fri' 4,'sat' 5,'sun' 6;
+
+// @each $weekDay in $days {
+//   $i: list.index($days, $weekDay) - 1;
+//   .calendar--start-day-#{$weekDay} .calendar__day:first-child {
+//     margin-left: calc($i * 100px + $i * 1px + 0.5px);
+//   }
+// }
```

Comment:
Do not leave unnecessary comments.
If this is not necessary, then it is better not to leave commented code